unumpy/linalg/__init__.py

```diff
@@ -0,0 +1 @@
+from ._multimethods import *
```

Comment:
Add `__all__`.

Reply:
I've added `__all__`. I wasn't sure if the import statement should stay or not.

Reply:
The `import` will stay... Probably move `__all__` to `_multimethods.py`